src/redisSeed.ts

```
@@ -0,0 +1,46 @@
+  try {
+    await redisClient.set("posts", JSON.stringify(inputPosts));
+    console.log(`Redis seeding done! Seeded ${inputPosts.length} posts.`);
+  } finally {
+    // Always attempt to close the client. If disconnect fails, surface the error
+    // after attempting to close (don't swallow fatal errors silently).
+    try {
+      await redisClient.quit();
+    } catch (err) {
+      console.error("Failed to disconnect Redis client:", err);
+      // rethrow so callers see the original failure if needed
+      throw err;
```

Biome (2.1.2)

[error] Unsafe usage of 'throw'.

'throw' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)